pkg/controller/perconaservermongodb/users.go

```diff
@@ -6,18 +6,16 @@ import (
 	"crypto/sha256"
 	"encoding/json"
 	"fmt"
-
+	api "github.com/percona/percona-server-mongodb-operator/pkg/apis/psmdb/v1"
+	"github.com/percona/percona-server-mongodb-operator/pkg/psmdb"
+	"github.com/percona/percona-server-mongodb-operator/pkg/psmdb/mongo"
```

Comment:
nitpick: imports should be grouped separately:

1. standard library
2. third party
3. our packages